src/dstack/_internal/core/models/repos/local.py

```diff
@@ -75,6 +79,7 @@ def write_code_file(self, fp: BinaryIO) -> str:
                 arcname="",
                 filter=TarIgnore(self.run_repo_data.repo_dir, globs=[".git"]),
             )
+        logger.debug(f"Code file size: {sizeof_fmt(fp.tell())} bytes")
```

Comment:
```suggestion
        logger.debug(f"Code file size: {sizeof_fmt(fp.tell())}")
```

sizeof_fmt should already emit B/Kib/MiB suffix so "bytes" is not necessary